system/logaccess/ElasticStack/ElasticStackLogAccess.cpp

```diff
@@ -262,6 +263,183 @@ const IPropertyTree * ElasticStackLogAccess::getESStatus()
     return performAndLogESRequest(Client::HTTPMethod::GET, "_cluster/health", "", "Target cluster health");
 }
 
+ bool ElasticStackLogAccess::healthReport(StringBuffer & report, LogAccessHealthReportOptions options)
+ {
+    try
+    {
+        report.appendf("\"ConnectionInfo\": { \"ConnectionString\": \"%s\" }", m_esConnectionStr.str());
+
+        report.append(", \"ConfigurationInfo\": { ");
+        if (m_pluginCfg)
+        {
+            StringBuffer configJSON;
+            toJSON(m_pluginCfg, configJSON, 0, JSON_Format|JSON_HideRootArrayObject);
+            report.appendf("\"ConfigurationTree\": %s", configJSON.str()); //json encode
```

Comment:
This is supposed to be more JSON appended to the report, not a string that contains the JSON configuration, correct? If it is the latter then it probably should be encoded.